includes/controller/guest_login_controller.php

```diff
@@ -20,7 +20,7 @@ function guest_register() {
   $nick = "";
   $lastname = "";
   $prename = "";
-  $age = "";
```

Comment:
Since the 'Alter' field in the User table is an integer, it cannot be set equal to an empty value